README.md

```diff
@@ -20,16 +20,12 @@ python setup.py develop
 ```
 
 ## Tests
-To run the tests in your virtualenv, execute:
-```shell
-python setup.py test
-```
-
-Alternatively, you can use `tox` or `detox` to run the tests against multiple versions of python:
+You can use `tox` or `detox` to run the tests against multiple versions of python:
 ```shell
 sudo dnf install python-detox
```

Comment:
is there any version for python 3?

Reply:
tox will automatically run against python 2.7 and 3.6. See in tox.ini `envlist = clean, py27, py36, report, flake8`

Reply:
cool